src/main/java/pl/agh/edu/iisg/io/vmms/vmmsbackend/controller/VMPoolController.java

```diff
@@ -43,46 +38,32 @@ public VMPoolController(VMPoolService vmPoolService) {
         return vmPoolService.findByDescriptionContaining(tag);
     }
 
+
+    @RequestMapping(value = "/update", method = RequestMethod.POST)
```

Comment:
Hmmm, nowy endpoint i nowy brak aktualizacji w README.md?
[technicznie rzecz biorąc jest to informacja o tym, że Admin Guide może na jutro nie powstać]